modules/140-user-authz/crds/authorizationrule.yaml

```diff
@@ -0,0 +1,121 @@
+                  description: |
+                    Users and/or groups to grant privileges.
+
+                    [Kubernetes API reference...](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#subject-v1-rbac-authorization-k8s-io)
+
+                    Pay attention to the following nuances if this module is used together with the [user-authn](https://deckhouse.io/documentation/v1/modules/150-user-authn/) module:
+                    - Use the user's `email` as the username to grant privileges to the specific user;
+                    - When specifying a group, make sure that the necessary groups are allowed to be received from the provider, i.e., they are defined in the corresponding custom resource [DexProvider](https://deckhouse.io/documentation/v1/modules/150-user-authn/cr.html#dexprovider).
+                  items:
+                    type: object
+                    required:
+                    - kind
+                    - name
+                    properties:
+                      kind:
+                        type: string
+                        enum: [User, Group, ServiceAccount]
+                        description: 'Type of user identification resource.'
+                        example: 'Group'
+                      name:
+                        type: string
+                        description: 'Resource name.'
+                        example: 'some-group-name'
+                additionalRoles:
+                  type: array
+                  description: |
+                    Additional roles to bind for subjects.
+
+                    This parameter is reserved for emergencies. Please, use the `accessLevel` parameter instead.
+
+                  x-doc-example: |
+                    ```yaml
+                    additionalRoles:
+                    - apiGroup: rbac.authorization.k8s.io
+                      kind: Role
+                      name: write-all
+                    - apiGroup: rbac.authorization.k8s.io
+                      kind: ClusterRole
+                      name: cluster-read-all
+                    ```
+                  items:
+                    type: object
+                    required:
+                    - apiGroup
+                    - kind
+                    - name
+                    properties:
+                      apiGroup:
+                        type: string
+                        description: 'apiGroup for users.'
+                        example: 'rbac.authorization.k8s.io'
+                        minLength: 1
+                      kind:
+                        type: string
+                        description: 'Kind of the role.'
+                        enum: [Role,ClusterRole]
+                        example: 'Role'
+                      name:
+                        type: string
+                        description: 'Name of the role.'
+                        minLength: 1
+                        example: 'admin'
```

Comment:
```suggestion
```
This is a security hole, and should be avoided for authorization rules. 
There is a way to add more access rules for basic roles now, and the cluster administrator should use it for customization https://deckhouse.io/documentation/v1/modules/140-user-authz/usage.html#customizing-rights-of-high-level-roles